src/styles/_popup.scss

```diff
@@ -28,6 +28,8 @@
 }
 
 .maputnik-feature-property-popup {
+  max-height: 500px;
```

Comment:
Hey @pathmapper, I think we can make this slightly more robust, at the moment if you resize the browser window height below `500px` this will suffer the same issue as before. Fortunately `100vh` in CSS is the height of the page, and we know that the toolbar is `40px` so I believe we can set to `max-height: calc(100vh - 40px - SOME_ADDTIONAL_VALUE);` where `SOME_ADDTIONAL_VALUE` is a additional amount to reduce the `max-height` by in order to display the popup arrow and a bit of padding. So for example

```
max-height: calc(100vh - 40px - 40px);
```

Reply:
Thanks a lot @orangemug, that's neat!

New demo:
https://1724-67726921-gh.circle-artifacts.com/0/artifacts/build/index.html